src/screens/BlaireHome/_private/SingleCard/SingleCard.tsx

```diff
@@ -0,0 +1,80 @@
+import type {ImageSourcePropType} from 'react-native';
+import * as React from 'react';
+import {useCallback, useEffect, useState} from 'react';
+import {Dimensions, Image, Pressable, View} from 'react-native';
+import Animated, {
+  runOnJS,
+  useAnimatedStyle,
+  useSharedValue,
+  withTiming,
+} from 'react-native-reanimated';
+
+import styles from './styles';
+
+const coverImage = require('screens/BlaireHome/assets/cover.png');
+
+const NUM_COLS = 4;
+const WIDTH = Dimensions.get('screen').width;
+
+type Props = {
+  cardImageSource: ImageSourcePropType;
+  onPress: () => void;
+  flipped: boolean;
```

Comment:
```suggestion
  isFlipped: boolean;
```
it's a nice-to-have for most booleans to be prefixed with `is` or something similar